src/pages/CropSidebar/CropSidebar.js

```diff
@@ -140,6 +140,20 @@ const CropSidebar = ({
     });
 
     const filtered = cropData?.filter((crop, n, cd) => {
+      const floodingFrequencyValue = crop.attributes.filter((a) => a.label === 'Flooding Frequency Tolerance')[0]?.values[0];
+      const getFloodingFrequency = () => {
```

Comment:
this function is confusing. i think i get what it's doing (returning the index of the highest drainage value associated with the string value) but it's not super obvious. can you either add comments to explain the logic and name the function more clearly or use a data structure that is more intuitive?